snap/hooks/install

```diff
@@ -21,15 +21,24 @@ esac
 export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/$MULTI_ARCH_PATH:$SNAP/usr/lib/$MULTI_ARCH_PATH"
 export LD_LIBRARY_PATH=$SNAP_LIBRARY_PATH:$LD_LIBRARY_PATH
 
+# get the values of $SNAP_DATA and $SNAP using the current symlink instead of
+# the default behavior which has the revision hard-coded, which breaks after
+# a refresh
+SNAP_DATA_CURRENT=${SNAP_DATA/%$SNAP_REVISION/current}
+SNAP_CURRENT=${SNAP/%$SNAP_REVISION/current}
```

Comment:
I couldn't find any instances of $SNAP being used in the delhi configurations files?

Reply:
Yes, again this was from my misapplied commit. $SNAP only needs to be replaced with the system management agent executor, which as you pointed out is an Edinburgh feature